src/herder/test/HerderTests.cpp

```diff
@@ -579,7 +578,7 @@ testTxSetWithFeeBumps(uint32 protocolVersion)
             auto tx1 = transaction(*app, account1, 1, 1, 100);
             auto fb1 = feeBump(*app, account2, tx1, 200);
             txSet->add(fb1);
-            auto tx2 = transaction(*app, account1, 2, 1, -1);
+            auto tx2 = transaction(*app, account1, 2, -1, 100);
```

Comment:
not sure I understand why you're using "-1" here: this causes tx1 and tx2 to have the same sequence number, and I think what you're testing is that fb2 doesn't have enough fee? Section name is confusing as there is no "malformed operation" involved here